tests/unit/lib/charms/vault_k8s/v0/test_vault_kv.py

```diff
@@ -390,6 +395,15 @@ def test_given_unit_leader_when_unit_joined_then_connected_event_fired_and_all_r
         assert isinstance(event, VaultKvConnectedEvent)
         assert app_relation_data["mount_suffix"] == self.harness.charm.interface.mount_suffix
 
+    @patch("test_vault_kv.VaultKvRequirerCharm._on_connected")
+    def test_given_unit_leader_when_refresh_event_then_connected_event_fired(
+        self, _on_connected
+    ):
+        self.setup_relation()[-1]
```

Comment:
We don't need the [-1] here, we're not assigning it to anything.

Reply:
Copy paste :D Thanks